docs/source/license.rst

```diff
@@ -1,7 +1,9 @@
-=======
-License
-=======
+.. _license:
+
+================
+:index:`License`
+================
 
 -  Apache-2.0
 -  Multiple licenses (LGPL, MIT, BSD, etc.) for third-party components.
--  See the `NOTICE <https://github.com/nexB/scancode-workbench/blob/develop/NOTICE>`__ file for more details.  *[The copyright date needs to be updated or removed.]*
+-  See the `NOTICE <https://github.com/nexB/scancode-workbench/blob/develop/NOTICE>`__ file for more details.  :guilabel:`[The copyright date needs to be updated or removed.]`
```

Comment:
We could use a ticket for this task as well.

Reply:
Ticket entered -- we have a few open questions and then I'll take care of this asap.